frontend/src/components/LandingPage/Weather/Weather.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './Weather.css'
+
+const weatherData = {
+    city: "Allen" + " ",
+    temperature: "65" + '\xB0F ',
+    picture: "/partly cloudy.png"
+};
+
+class Weather extends React.Component {
+    render() {
+        return(
```

Comment:
Use [props destructuring](https://dev.to/arnas/destructuring-react-props-for-the-cleaner-code-293) so that you do not have to keep typing the variable name several times.

`const { city, temperature, picture } = weatherData;`

With the props destructuring line above, simply type the property names. For instance, `<b>{city}</b>` instead of `<b>{weatherData.city}</b>`.